a8c-files.php

```diff
@@ -1156,6 +1156,59 @@ function a8c_files_maybe_inject_image_sizes( $data, $attachment_id ) {
 
 	return $data;
 }
+/**
+ * Filter dimensions for images to return a proper image size and srcset values
+ *
+ * @param bool   $value         The filtered value, defaults to `true`.
+ * @param string $image         The HTML `img` tag where the attribute should be added.
+ * @param string $context       Additional context about how the function was called or where the img tag is.
+ * @param int    $attachment_id The image attachment ID.
+ */
+function a8c_files_maybe_inject_dimensions( $value, $image, $context, $attachment_id ) {
```

Comment:
Any reason not to use type declarations on this parameters?

Reply:
Good question!   We're not really using most of these since we're just returning `true` for the value in this filter and we're not returning any of them either, so I wasn't sure if that would be helpful in this case.  Would you suggest still adding them?  I also wasn't sure if I should pass them all even if we're not using them?

Reply:
If you're not using them, then there is no need to support them in the function signature, unless a param to the right is being used.

However, since you need `$attachment_id`, then they all need to be passed.

The non-first types should be known, as they are from core code, but the first type may have been filtered at an earlier priority into a different type, so although it's not being used here, the advantage of using types here is that it becomes an extra check in case some other code is doing something wrong. It's also good for IDEs and static analysis tools, like Psalm or Phpstan

Adding the return type also has no drawbacks.